doc/api/esm.md

```diff
@@ -219,7 +233,15 @@ The `"main"` field can point to exactly one file, regardless of whether the
 package is referenced via `require` (in a CommonJS context) or `import` (in an
 ES module context).
 
-### Package Exports
+[Package Exports][] provide an alternative to `"main"` where the package main
+entry point can be defined while also encapsulating the package, preventing any
+other entry points besides those defined in `"exports"`. If package entry points
+are defined in both `"main"` and `"exports"`, the latter takes precedence in
+versions of Node.js that support `"exports"`. [Conditional Exports][] can also
+be used within `"exports"` to define different package entry points per
+environment.
```

Comment:
Can we combine this into the top paragraph on exports? The main thing we need seems to be just to mention the precedence, which should be as simple as just adding something like "The exports field, which takes higher precedence than main, ..." or similar.